### PR TITLE
Common doc PR for all the modules and drivers

### PR DESCRIPTION
**Pull request by shanmydell**
# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [x] Have you tested whether the hyperlinks are working properly?
- [x] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/authorization/_index.md [hunk 1]
```diff
@@ -68,6 +68,7 @@ CSM for Authorization consists of 2 components - the Authorization sidecar and t
 | Authorization Sidecar Image Tag | Authorization Proxy Server Version |
 | ------------------------------- | ---------------------------------- |
 | dellemc/csm-authorization-sidecar:v1.0.0 | v1.0.0, v1.1.0 |
+| dellemc/csm-authorization-sidecar:v1.2.0 | v1.0.0, v1.1.0, v1.2.0 |
```
Comment (gallacher) [inline content/docs/authorization/_index.md:71]: Please remove v1.0.0 from this new entry

File: content/v1/FAQ/_index.md [hunk 2]
```diff
@@ -33,13 +33,13 @@ The umbrella repository for every Dell Container Storage Module is: [https://git
...
+| CSM Module        | CSI PowerFlex v2.2 | CSI PowerScale v2.2 | CSI PowerStore v2.2 | CSI PowerMax v2.2 | CSI Unity XT v2.2    |
 | ----------------- | -------------- | --------------- | --------------- | ------------- | --------------- |
-| Authorization v1.0| ✔️              | ✔️               | ❌              | ✔️             | ❌            |
+| Authorization v1.2| ✔️              | ✔️               | ❌              | ✔️             | ❌            |
 | Observability v1.0| ✔️              | ❌              | ✔️               | ❌            | ❌            |
-| Replication   v1.0| ❌             | ❌              | ✔️               | ✔️             | ❌            |
-| Resilency     v1.0| ✔️              | ❌              | ❌              | ❌            | ✔️             |
-| CSM Installer v1.0| ✔️              | ✔️               | ✔️               | ✔️             | ✔️             |
+| Replication   v1.2| ❌             | ✔️              | ✔️               | ✔️             | ❌            |
+| Resilency     v1.2| ✔️              | ❌              | ❌              | ❌            | ✔️             |
```
Comment (hoppea2) [inline content/v1/FAQ/_index.md:41]: Resiliency Release will be 1.1

**Review by coulof — APPROVED** [review]

File: content/v1/FAQ/_index.md (hunk 2 above)
Comment (shanmydell) [reply content/v1/FAQ/_index.md:41]: @walker2: This is a wrong edit for v1, I have stashed this not sure how it got checked in. Will revert the changes.

File: content/docs/authorization/_index.md (hunk 1 above)
Comment (shanmydell) [reply content/docs/authorization/_index.md]: Addressed

**Review by prablr79 — APPROVED** [review]

**Review by francis-nijay — APPROVED** [review]

**Review by walker2 — APPROVED** [review]

**Pull request merged**